State docfix (#399)
pytorch/ignite: docs/source/concepts.rst

    trainer.add_event_handler(Events.ITERATION_COMPLETED, on_iteration_completed)

> There are many ways of creating a `process_function`, this is why Ignite provides output_transform parameter for its 

Reviewer: I would rephrase this as 
```suggestion
Since there is no restrictions on `process_function`, Ignite provides `output_transform` argument for its 
```
@anmolsjoshi what do you think ?

Author: It reads better!

> :class:`~ignite.metrics` and :class:`~ignite.handlers`. `output_transform` is a function used to transform `engine.state.output` for intended use. Below we'll see different types of `engine.state.output` and how to transform them.

Reviewer: ```suggestion
:class:`~ignite.metrics` and :class:`~ignite.handlers`. Argument `output_transform` is a function used to transform `engine.state.output` for intended use. Below we'll see different types of `engine.state.output` and how to transform them.
```

Author: Reads better!
